ResearchKitOnFhir/researchKitOnFhir/ViewControllers/SurveyListViewController.swift

```diff
@@ -0,0 +1,244 @@
+//
+//  SurveyListViewController.swift
+//  researchKitOnFhir
+//
+
+import UIKit
+import ResearchKit
+import SMART
+
+class SurveyListViewController: UIViewController {
+    
+    var smartClient: Client?
+    var didAttemptAuthentication = false
+    static var questionnaireList = [Int: QuestionnaireType]()
+    var todoQList = [QuestionnaireType]()
+    var completeQList = [QuestionnaireType]()
+    let tableView = UITableView()
+    static var QList = [QListCategory]()
+    
+    struct Cells {
+        static let buttonCell = "buttonCell"
```

Comment:
You can also add a static property to the CustomTableViewCell for the reuse identifier. It's a little bit cleaner to have everything related to that cell in the cell class itself.